pkg/volume/rbd/rbd_util.go

```diff
@@ -269,7 +270,11 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) error {
 
 		// fence off other mappers
 		if err = util.fencing(b); err != nil {
-			return fmt.Errorf("rbd: image %s is locked by other nodes", b.Image)
+			if err.Error() == rbdCmdErr {
+				return fmt.Errorf("rbd: rbd cmd not found")
+			} else {
+				return fmt.Errorf("rbd: image %s is locked by other nodes", b.Image)
+			}
```

Comment:
nit: i would wrap this whole block into a function like `return rbdErrors()` or something and use it everywhere